impeller/renderer/backend/vulkan/context_vk.cc

```diff
@@ -184,6 +184,17 @@ void ContextVK::Setup(Settings settings) {
 
   vk::InstanceCreateInfo instance_info;
   if (caps->AreValidationsEnabled()) {
+    std::stringstream ss;
+    ss << "Enabling validation layers, features: [";
+    for (const auto& validation : enabled_validations) {
+      ss << vk::to_string(validation) << " ";
+    }
+    ss << "]";
+    FML_DLOG(ERROR) << ss.str();
+#if !IMPELLER_ENABLE_VULKAN_VALIDATION_LAYERS && FML_OS_ANDROID
+    VALIDATION_LOG << "Attempting to use validation layers without "
```

Comment:
I don't think VALIDATION_LOG is right here - unless I'm misunderstanding and it's an error to end up in this state (although it seems like there are multiple ways now to turn on validation and I'm not clear on which path(s) are allowed or not).

Reply:
Yep, it's an error.  We want people to be using the validation layers that we are building.  There is no reason to turn on validation layers without building and packaging up validation layers.  Jonah ran into this the other day where he thought just turning on the environment variable was sufficient then had to go through the rigamarole to package prebuilt layers.

Reply:
I updated it.  I made it a bit stronger and also made sure to mention `gn` (as opposed to runtime arguments).

Reply:
(sorry, uploaded now, i accidentally committed the local hack i need to make scenario_app work on my machine instead)

Reply:
VALIDATION_LOG  isn't fatal by default and won't show up in logcat unless fatal validations are enabled

Reply:
So, this will crash in a way that is hard to know what is going on if the runtime flag is on, but it wasn't built to support it.  Sounds like we want an `FML_LOG(ERROR)` instead.

Reply:
Switched both things to `FML_LOG(ERROR)`. There is no need to use `DLOG` since using validation layers should never happen in the wild anyways.  `ERROR` since we know it will probably crash.